platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/AbstractMigratable.java

```diff
@@ -21,35 +21,17 @@
  * This class handles the boilerplate code that all {@link org.codice.ddf.migration.Migratable}'s
  * need to implement.
  */
-public abstract class AbstractMigratable implements Migratable {
+public abstract class AbstractMigratable {
 
     protected final String description;
 
-    protected final boolean isOptional;
-
     /**
      * Constructor
      *
      * @param description a short description of what this {@link org.codice.ddf.migration.Migratable} does.
-     * @param isOptional is the exported data from this {@link org.codice.ddf.migration.Migratable} optional or required
      */
-    public AbstractMigratable(@NotNull String description, boolean isOptional) {
+    public AbstractMigratable(@NotNull String description) {
         notNull(description, "description cannot be null");
         this.description = description;
-        this.isOptional = isOptional;
-    }
-
-    /**
-     * @return a short description describing the purpose of this {@link org.codice.ddf.migration.Migratable}
-     */
-    public String getDescription() {
```

Comment:
@Lambeaux, maybe we should keep this abstract class since we'll need one anyways when we start using the `Describable` interface. Thoughts?